scripts/utility.py

```diff
@@ -403,25 +403,25 @@ def create_outside_cat(Cat, status, backstory):
         TODO: DOCS
         """
         suffix = ''
-        if 'rogue' in backstory:
+        if backstory in Cat.backstory_categories["former_rogue_backstories"]:
             status = 'rogue'
-        elif backstory in ['ostracized_warrior', 'disgraced', 'retired_leader', 'refugee',
-                         'tragedy_survivor', 'disgraced2', 'disgraced3', 'refugee5']:
-            status = "former clancat"
+        elif backstory in Cat.backstory_categories["former_clancat_backstories"]:
+            status = "former Clancat"
         if status == 'kittypet':
             name = choice(names.names_dict["loner_names"])
         elif status in ['loner', 'rogue']:
             name = choice(names.names_dict["loner_names"] +
                                  names.names_dict["normal_prefixes"])
-        elif status == 'former clancat':
+        elif status == 'former Clancat':
```

Comment:
Is 'former Clancat' a valid status?
I thought 'exiled' is used for ex-Clancats.